python/thunder/factorization/ica.py

```diff
@@ -22,8 +22,10 @@ class ICA(object):
         Number of independent components to estimate
 
     svdMethod : string, optional, default = "auto"
-        Which SVD method to use. If set to 'auto',
-        will select preferred method based on dimensionality.
+        Which SVD method to use.
+        If set to 'direct', will compute the SVD directly.
```

Comment:
replace "directly" with "direct gramian matrix estimation and eigenvector decomposition" here and in the other one